buildscripts/docker/docker_release/README.md

```diff
@@ -0,0 +1,75 @@
+# Bodo DockerHub Image
+
+Bodo publishes a new DockerHub image with each release, enabling users to run Bodo inside Docker containers.
+
+The image includes two example notebooks to help you get started with Bodo:
+    - quickstart.ipynb: Demonstrates how to use Bodo and BodoSQL for Pandas and data transformation applications.
+    - pi.ipynb: Implements a Monte Carlo simulation to estimate the value of pi.
+
+NOTE: 
+- This image is compatible with Linux x86 and macOS x86 platforms only.
+- The image is intended for testing and learning purposes and should not be used in production.
+
+## Prerequisites
+Before you begin, ensure the following:
+
+1. The latest version of Docker is installed on your machine.
+2. The Docker daemon is running (e.g., via Docker Desktop).
+
+## Download and Run the Container
+
+The Bodo image is available on DockerHub:
+https://hub.docker.com/r/bodoai/bodoai_image
```

Comment:
Can we discuss the name a bit? I think a more descriptive name might be more benefitial, like `bodoai/jupyterlab`. Cause the name `bodoai_image` is a bit repetitive and unnecessary. Plus, we may want to have more image types in the future, like a minimal image for example without jupyter or a perf image with better MPI, etc.

Reply:
I'm ok. with changing name. Let me know what it should be.
I don't think it should be `jupyterlab` though as it can be used without it.

Reply:
I agree that jupyterlab is a good name, but if you don't like it maybe bodo_example? If we go with that I think we need to setup a minimal image too

Reply:
Generally I've seen images named by what features or special software is installed. For example, I've seen like `spark-sql`, spark-jupyter` etc based on what additional functionality is supported. So even though you dont need to use `jupyterlab` with the image, because its installed I think it should be named that.